src/polymer/behavior-scanner.ts

```diff
@@ -290,13 +290,24 @@ class BehaviorVisitor implements Visitor {
  * gets the expression representing a behavior from a node.
  */
 function behaviorExpression(node: babel.Node): babel.Node|null|undefined {
+  if (babel.isVariableDeclarator(node)) {
```

Comment:
It would be nice to have test coverage for all these syntaxes.

Reply:
All of these did show up in our various tests, though not all in the behaviors tests

Reply:
But no other tests changed so how did this work before?

Reply:
That's a damn good question. I added an explicit test of a default exported behavior.